pages/coaching-dashboard.tsx

```diff
@@ -38,6 +44,19 @@ const coachingDashboard = () => {
       },
     }
   );
+  useQuery<FetchUserRoleData>(FETCH_USER_ROLE, {
+    variables: {
+      _id: user.uid,
+    },
+    if(loading) {
+      return <div>Loading...</div>;
+    },
+    onCompleted: (roleData) => {
+      if (roleData.users[0].userRole.value === "student") {
```

Comment:
Can we change this to check if the value is not equal to "coach" since @kwiedenhaupt is adding a new freemiun role and we wouldn't want them to have access to the coaching dashboard either.

Reply:
oh good point! will do.